doc/sphinx/source/project/introduction.rst

```diff
@@ -4,176 +4,86 @@
+
+Second approach is to deliever performance via software optimization, and nGraph Compiler stack is our solution to the second apporach. We created nGraph compiler stack not only to provide software acceleration for the upcoming deep learning ASICs but also to unlock massive performance boost for existing hardware targets such as CPUs and GPUs. Additionally, it aims to address the scalability issue with kernel libraries which is a popular solution to accelerate deep learning performance.  
 
 Motivations
 ===========
 
-Kernel libraries do not support graph-level optimizations
----------------------------------------------------------
+The current state of art software solution for speeding up deep learning computation is to integrate kernel libraries such as Intel MKL-DNN and NVidia's CuDNN into deep learning frameworks. These kernel libraries offer runtime performance boost on specific hardware targets through highly optimized kernels and operator level optimizations.
 
-A framework designed for training using GPUs requires integration with a kernel 
-library unique to that vendor's hardware. For example, after integration, a 
-kernel library can run operations that it is "familar" with optimally; however, 
-the graph itself within any larger :term:`NN` won't be optimal.
+However, kernel libraries have three main problems: 
 
-.. _figure-A:
+1. Kernel libraries do not support graph level optimizations
+2. Framework integration of kernel libraries does not scale
+3. There are too many kernels to write, and they require expert knowledge 
 
-.. figure:: ../graphics/framework-to-kernel-lib.png
-   :width: 555px
-   :alt: 
-
-   Figure A: Lack of graph-level optimization makes framework-to-kernel library
-   integration enormously inefficient. The computation graph above represents 
-   the computation: "A plus B times C".
-
-
-.. _figure-B:
+nGraph Compiler stack is designed to address the first two problems. nGraph applies graph level optimizations by taking computational graph from deep learning frameworks and reconstructing it with nGraph IR (Intermediate Representations). nGraph IR centralizes computational graphs from various frameworks and provides a unified way to connect backends for targetted hardware. To address the third problem, nGraph Compiler stack is integrated with PlaidML to generate code in LLVM, OpenCL, OpenGL, Cuda and Metal with low level optimizations automatically applied. 
```

Comment:
The nGraph Compiler stack addresses the first two issues.  It allows for both automatic and manual graph-level optimizations via the nGraph IR, which centralizes... 

typo on targeted

Reply:
Long term, we hope to address 3 with things like PlaidML.